### ISSUE TITLE
Don't allow Ctrl+C to kill the outermost command processor

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#900]
_From [robertco...@yahoo.com](https://code.google.com/u/106635827517471658382/) on January 21, 2013 11:29:16_

I have the habit of pushing "Ctrl+C" over and over again without looking if I want to terminate everything and get back to the "base console" prompt.

With cmd.exe - this is fine because pushing "Ctrl+C" on an outer instance will do nothing.

However, with ConEmu , Ctrl+C also prompts for and can kill the outermost cmd.exe process. This seems awkward to me. If I wanted to close the window - I would use the hotkey Alt+F4. Also if cmd.exe is run inside of cmd.exe - then I want Ctrl+C (and its confirmation) to only show if I am in a nested cmd.exe instance.

In short, Ctrl+C should not have the ability to be able to terminate the window. Ctrl+C should only be able to terminate subprocesses running INSIDE the window.

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=900_

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on January 21, 2013 12:14:55_

Unclear. ConEmu 's Ctrl+C does not "kill" process. It works exactly as in real console.
May be video screencast, settings, or some more explanations will bring to understanding.

**Comment by ConEmu** [comment]
_From [robertco...@yahoo.com](https://code.google.com/u/106635827517471658382/) on January 21, 2013 13:29:13_

Oh, solved. I had "Ctrl+C" bound to "Terminate active process in current console" from an old ConEmu .xml. I removed the "Ctrl+C" binding in "Keys and Macros" and now Ctrl+C is being handled by cmd.exe directly with the desired behavior.

This can be closed.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on January 21, 2013 13:49:12_

**Status:** Done
